### ISSUE TITLE
Create recurring reminders

### ISSUE THREAD

**Issue by Half-Shot** [maubot/reminder#6]

**Comment by scottwallacesh** [comment]
Perhaps with a simple `... every <x> <m|h|d|w|y>` format to start with?

**Comment by joao-p-marques** [comment]
Yes. https://github.com/anoadragon453/matrix-reminder-bot already does that. It would be nice to incorporate that here. :+1:

**Comment by pullopen** [comment]
If anyone is still working on this...

**Comment by NicolasDerumigny** [comment]
I would very much like it to be done, too

**Comment by roughnecks** [comment]
Me too, so I can ditch matrix-reminder-bot.

**Comment by MxMarx** [comment]
I've added recurring reminders and integrated [matrix-reminder-bot's](https://github.com/anoadragon453/matrix-reminder-bot/tree/master) features as well as some others in this [this fork](https://github.com/MxMarx/reminder). I'm not sure if it is eligible to be included in the plugin list because it requires two libraries that aren't currently included in the maubot docker image: [APScheduler](https://github.com/agronholm/apscheduler) and [dateparser](https://github.com/scrapinghub/dateparser)  (and optionally [cron_descriptor](https://github.com/Salamek/cron-descriptor), [arrow](https://github.com/arrow-py/arrow)). It could do with a bit more testing, but it's been working for me at least!
